packages/palette_generator/lib/palette_generator.dart

```diff
@@ -150,6 +172,28 @@ class PaletteGenerator with Diagnosticable {
       'The encoding must be RGBA with 8 bits per channel.',
     );
 
+    // If we want to use an isolate, make a recursive compute call. We do it
```

Comment:
Who is "we" here? You've used it twice, and it does not appear possible for it to be referring to the same "we" each time.

For this reason, comments are generally much easier to understand when they use clear subjects rather than "we".

Reply:
Fixed in 4992c3b98, it's just a personal style I've picked up. I guess it's written as if I'm explaining the code to someone sitting next to me. I understand how it could be ambiguous though :)